clients/venice-client/src/main/java/com/linkedin/venice/fastclient/ClientConfig.java

```diff
@@ -84,6 +84,12 @@ public class ClientConfig<K, V, T extends SpecificRecord> {
    * route requests to the correct server/partition
    */
   private final GrpcClientConfig grpcClientConfig;
+  /**
+   * The time window used to calculate user traffic and corresponding long tail retry budget. The default value is 5
```

Comment:
The javadoc mentions 5mins and the following config is 1min:
```
    private long longTailRetryBudgetEnforcementWindowInMs = 60000; // 1 minute
```
Are these two related?

Reply:
Yes, good catch. FIxed.